cdap-client-tests/src/test/java/co/cask/cdap/client/common/ClientTestBase.java

```diff
@@ -57,6 +58,9 @@
   public static final SingletonExternalResource STANDALONE = new SingletonExternalResource(new StandaloneTester());
 
   @ClassRule
+  public static final TestConfiguration CONFIG = new TestConfiguration("explore.enabled", "true");
```

Comment:
No, but since StandaloneTester is also a `TestRule`, I had to put it like this:
```java
@ClassRule
  public static final SingletonExternalResource STANDALONE = new SingletonExternalResource(new StandaloneTester());

  @ClassRule
  public static final TestConfiguration CONFIG = new TestConfiguration("explore.enabled", "true");
```